### DISCUSSION TITLE
Prodigy

### DISCUSSION THREAD

**Discussion by Zonclp** [Prodigy-Hacking/ProdigyMathGameHacking#1220] [(A) Anything]
I Don't see why people/kids want to hack a game that's so simple and easy if prodigy that hard to play then put your grade level on 
1st grade There's no need To Hack

**Comment by donaldli2020** [comment]
it's fun

**Reply by Zonclp** [reply]
It's fun huh?  the hacks don't even work I tested out myself so how is it fun? sir

**Reply by donaldli2020** [reply]
ill fix w/ pull request unless someone else does

**Reply by LeoBadeaux** [reply]
> It's fun huh? the hacks don't even work I tested out myself so how is it fun? sir

yeah it's fun, we like messing around with this stuff and decided to make it public. the hacks are not always going to be up but nobody is being paid and with a post like that why would you care if the hacks are down?

**Reply by MisakeTheRat** [reply]
> It's fun huh? the hacks don't even work I tested out myself so how is it fun? sir

Hold on so your mad because it didn't work for you? Also you put "I Don't see why people/kids want to hack a game that's so simple and easy if prodigy that hard to play then put your grade level on
1st grade There's no need To Hack" AND YOU TRIED IT like bruh your logic is fucking dogshit let people have fun damn.

**Comment by jhatton2027** [comment]
:1

On Wed, Dec 8, 2021 at 9:20 AM Zonclp ***@***.***> wrote:

> It's fun huh? the hacks don't even work I tested out myself so how is it
> fun? sir
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/discussions/1220#discussioncomment-1773375>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATNYK4PETAA4CNWBQP4CFHTUP5ZSLANCNFSM5JTEVNMQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

-- 
NOTICE TO RECIPIENT: This communication and any response to it may 
constitute a public record, and therefore, may be available upon request in 
accordance with Iowa public records law, Iowa Code chapter 22.

**Comment by S0vereignity** [comment]
we hack because we have free wills and can do anything we want, like Jevil.
